common/lib/xmodule/xmodule/capa_module.py

```diff
@@ -139,36 +356,6 @@ def display_name_with_default(self):
 
         return self.display_name
 
-
-class CapaDescriptor(CapaFields, RawDescriptor):
-    """
-    Module implementing problems in the LON-CAPA format,
-    as implemented by capa.capa_problem
-    """
-    INDEX_CONTENT_TYPE = 'CAPA'
-
-    module_class = CapaModule
-    resources_dir = None
-
-    has_score = True
-    show_in_read_only_mode = True
-    template_dir_name = 'problem'
-    mako_template = "widgets/problem-edit.html"
-    js = {'js': [resource_string(__name__, 'js/src/problem/edit.js')]}
-    js_module_name = "MarkdownEditingDescriptor"
```

Comment:
This is used by `shim_xmodule_js()`. Have changed `shim_xmodule_js()` to accept the module name as a parameter and passed in from `studio_view()`.